src/middlewared/middlewared/alert/base.py

```diff
@@ -190,10 +190,18 @@ def format_alerts(product_name, hostname, node_map, alerts, gone_alerts, new_ale
         return text + "This is a test alert"
 
     if new_alerts:
-        text += "New alerts:\n" + "".join(["* %s\n" % format_alert(alert, node_map) for alert in new_alerts]) + "\n"
+        if len(gone_alerts) == 1:
+            text += "New alert"
+        else:
+            text += "New alerts"
+        text += ":\n" + "".join(["* %s\n" % format_alert(alert, node_map) for alert in new_alerts]) + "\n"
 
     if gone_alerts:
-        text += "Gone alerts:\n" + "".join(["* %s\n" % format_alert(alert, node_map) for alert in gone_alerts]) + "\n"
+        if len(gone_alerts) == 1:
+            text += "The following alert has been cleared"
```

Comment:
If it is immediately following, the reader can see that.
```suggestion
            text += "This alert has been cleared"
```